src/App.js

```diff
@@ -1,27 +1,27 @@
-import logo from './logo.svg';
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import AddHouse from './components/AddHouse';
+import HouseList from './components/HouseList';
+import ReserveList from './components/ReserveList';
+import MyReservations from './components/MyReservations';
+import Sidebar from './components/Sidebar';
 import './App.css';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <div className="sidebar-container">
+        <Sidebar />
+      </div>
+
+      <div className="main-container">
+        <Routes>
+          <Route path="/" element={<HouseList />} />
+          <Route path="/add-house" element={<AddHouse />} />
+          <Route path="/reserve-list" element={<ReserveList />} />
+          <Route path="/my-reservations" element={<MyReservations />} />
+        </Routes>
```

Comment:
- [ ] Please add the exact links to your routes are stated in your task description : 
- Houses
- Reserve*
- My Reservations
- Add House
- Delete House*